app/controllers/contacts/new.js

```diff
@@ -0,0 +1,133 @@
+import Controller from '@ember/controller';
+import { task } from 'ember-concurrency-decorators';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { A } from '@ember/array';
+
+export default class ContactsNewController extends Controller {
+  TITLES_ARRAY = ['Basis informatie', 'Ontvangers', 'Contactgegevens'];
+
+  @tracked step = 0;
+  @tracked isManualInput = '';
+  @tracked showContactItemModal = false;
+  @tracked showConfirmationModal = false;
+  @tracked selectedContact;
+
+  @tracked contacts = A([]);
+
+  get hasPrevious() {
+    return this.step > 0;
+  }
+
+  get hasNext() {
+    return this.step < this.TITLES_ARRAY.length - 1;
+  }
+
+  get isLast() {
+    return this.step === this.TITLES_ARRAY.length - 1;
+  }
+
+  get title() {
+    return this.TITLES_ARRAY[this.step];
+  }
+
+  get progress() {
+    return this.step * 100 / this.TITLES_ARRAY.length;
+  }
+
+  get isDisabledNextStep() {
+    if (this.step === 0) {
+      return !this.model.name;
```

Comment:
De [`isPresent` of `isBlank`](https://api.emberjs.com/ember/release/functions/@ember%2Futils/isPresent) utils zijn hier geschikter om values met enkel whitespaces niet als `true` te aanzien.